go/README.md

```diff
@@ -1 +1,190 @@
-# Go
\ No newline at end of file
+# Go Code Conventions
+
+## Modules
+
+go mod init bitbucket.org/aanzeeonline/[yourprojectname]
+
+Working with a local fork of a module, put an extra line in the go.mod:
+// replace bitbucket.org/aanzeeonline/libgoaz => ../LibGOAZ
+(and remove the comments)
+
+Before pushing the final update to git, make sure that the fork isn't in use anymore
+
+We need to investigate, if a vendor option (go mod vendor) is usable.
+(The upcoming 1.13 will have the final module solution.)
+"go mod vendor puts them in a vendor folder inside your project, running go build -mod vendor
+uses the dependency put in the vendor folder. This further isolates the project from the
+globally installed dependencies."
+
+## Project structure for an API / Web application
+
+root folder
+- holds various files for building
+/config
+main.go
+/dist
+- where the binary is placed
+/src
+	init.go
+	- bootstrap of the application
+	- only needed for an application (not for a library)
+	- holds the start/stop/init/config
+    /web holds the routing initialization
+    /helpers
+	- project specific
+	/middlewares (if they exist in the project)
+    /datamodels
+	/business (separation of concerns)
+    	/domain(s)
+            /handlers
+            /queries (if you work with a database)
+            /repositories
+            /services
+            /validators
+
+## 
```

Comment:
Looks like in the [last commit](https://github.com/AanZee/conventions/commit/3dac6b270d92db966ee286235279cfff22c3177c#diff-8df3ecdc89a1dca5fbf7dfa5c0a71c91L36) the title 'Makefile' was removed (intentionally?) and an empty title ("##") was added. I would suggest adding a title or removing the empty title :)

Reply:
fixed